src/StatisticalAnalyser.java

```diff
@@ -125,4 +125,16 @@ public double pathWeightMean(ArrayList<ArrayList<Path>> paths) {
         stats[1] = Math.sqrt(acc/(double)nPaths); // desvio padrao
         return stats;
     }
+
+    // Calculate routing distance -> all paths lengths / #paths
+		public double averageRoutingDistance(ArrayList<ArrayList<Path>> paths) {
+			double routingDistance = 0.0;
+
+			for (ArrayList<Path> path : paths)
+				routingDistance += path.size();
+
```

Comment:
You can remove this blank line. The comment line coming after will split the two code portions.